presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the ``http-server.https.excluded-cipher`` config
+  property to exclude non FS cipher suites. Specifically, all
+  ciphers without forward secrecy, and all ciphers that use the RSA key exchange
+  algorithm are excluded by default. Consequently, TLS 1.0 or TLS 1.1 are no
+  longer supported by default. The ``http-server.https.excluded-cipher`` config property
+  can be set to empty string to restore the old behavior.
+* Add :func:`ST_GeomFromBinary` and :func:`ST_AsBinary` functions that convert
+  geometries to and from Well-Known Binary format.
+* Remove the ``verbose_stats`` session property, and rename the ``task.verbose-stats``
+  configuration property to ``task.per-operator-cpu-timer-enabled``.
+* Remove the ``experimental.enable-new-stats-calculator`` configuration property.
+* Improve query planning performance for queries containing multiple joins
+  and a large number of columns (:issue:`11196`).
+* Allow running on a JVM from any vendor that meets the functional requirements.
+
+Hive Connector Changes
+----------------------
+
+* Fix regression in 0.210 that causes query failure when writing to certain ORC/DWRF files.
+  The failure occurs when attempting to convert a highly dictionary compressed column
+  to direct encoding that would exceed max configured stripe size. TODO: wenleix
+* Fix coordinator OOM when a query scans many partitions of a Hive table (:issue:`11322`).
+* Improve readability of columns, partitioning, and transactions in explain plains.
+
+Resource Group Changes
+----------------------
+
+* Add documentation for new resource group scheduling policies.
+* Add built-in :doc:`file based property manager </admin/session-property-managers>`
+  to automate the setting of session properties based on query characteristics.
```

Comment:
Should this be under "Resource Group Changes"? Apparently this can be used independently from resource groups.